### PR TITLE
Fixed order of decorators in documentation

### PR DESCRIPTION
**Pull request by K-Mistele**
The CodeLighthouse `error_catcher` decorator must be inside the `app.route` decorator

### CODE REVIEW

File: docs/python/py_sdk_docs.md [hunk 1]
```diff
@@ -71,7 +71,8 @@ Once you have configured the SDK, it's super easy to use! Simply use the CodeLig
 
 ### Using the Error Catcher Decorator
 Each decorator only applies to the one function defined directly below it. In the decorator, specify the email address of the user in your organization who should receive the notification. 
-
+Note that for frameworks like Flask that use decorators to denote a route, the Codelighthouse `error_catcher` decorator 
+**must** be the innermost decorator!
```
Comment (willtraweek) [inline docs/python/py_sdk_docs.md:75]: This technically isn't true -- the reason that flask requires CLH to be inside of it is because Flask's decorator sets up the context for what happens when the function is called after app.run() has been called.  It's a symptom of how WSGI manages processes.

This isn't a problem that will happen with 99.9% of decorators.  Technically we want it to actually be the outer-most decorator since that way it will also capture issues that the more inner decorators will throw.

File: docs/python/py_sdk_docs.md [hunk 2]
```diff
@@ -81,6 +82,8 @@ def some_function():
 
 ### Adding Additional Users
 You can invite additional users to your organization in your admin panel on the [user management page](https://codelighthouse.io/admin/users). Note that each payment plan only comes with a fixed number of users, and that adding additional users past that number will cost more. Please refer to our [pricing guide](https://codelighthouse.io/#pricing) for more information.
+Once you've invited users, you can use their email addresses in decorators to assign them as code owners, and notifications
+will be sent to them.
```
Comment (willtraweek) [inline docs/python/py_sdk_docs.md]: I think we need to add the part about them needing to accept the invite.  Since the code won't work effectively without that

**Review by willtraweek — CHANGES_REQUESTED** [review]

File: docs/python/py_sdk_docs.md (hunk 2 above)
Comment (K-Mistele) [reply docs/python/py_sdk_docs.md]: @willtraweek fixed

File: docs/python/py_sdk_docs.md (hunk 1 above)
Comment (K-Mistele) [reply docs/python/py_sdk_docs.md:75]: @willtraweek what should I do about it? hard to know which ones we do vs don't want to come before

Comment (willtraweek) [reply docs/python/py_sdk_docs.md:75]: Not necessarily -- the only ones that need to be outside of the codelighthouse one are decorators that change what triggers the function.  Django, Flask, and Pyramid are the only ones I can think of ATM.

As for direct changes, we shouldn't say it **must** be the innermost decorator.  Rather, I would rephrase the above to the below:
```
Note that the CodeLighthouse decorator must be inside of decorators for web frameworks (Flask, Django, Pyramid)
```
